PythonPonies/bangazon_cli
# Ike fully documented modules

# Description
feature of command line interface for completing an order

## Number of Fixes
2: tests all pass; full documentation

## Related Ticket(s)


## Problem to Solve
passes tests.  Ties CLI to database functions.

## Proposed Changes

Complete orders via the command line interface

## Expected Behavior
at root run the command line interface and choose the 5th option.  Complete an order to cancel an order that is active.

## Steps to Test Solution

1. git fetch branch
2. Move to root folder:
run  `python bangazon.py 5`

Navigate through directions:

If you choose a valid payment option (choose 4 for RFK, the active user for testing purposes)
see the 'victory message' that tells you the order is complete and print screen

If you don't complete the order:
 you should see a statement about the proper way to confirm an order.

NEXT
drop database and load new database via `python main.py`
switch to test folder:
run all tests via `python runtests.py`
## Testing

[x ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x ] I certify that all existing tests pass

## Documentation
[ x] There is new documentation in this pull request that must be reviewed..
[x ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
